agent/engine/dockerclient/versionsupport_unix.go

```diff
@@ -40,6 +40,10 @@ func getAgentVersions() []DockerVersion {
 		Version_1_23,
 		Version_1_24,
 		Version_1_25,
+		Version_1_26,
```

Comment:
Did you tested that these capabilities are registered as expected, `com.amazonaws.ecs.capability.docker-remote-api.1.[26,27,28,29]` for the container instance attributes?

Reply:
Yes, the capabilities for the api versions(if supported) are registered as expected.